### PR TITLE
Add oso w3id

### PR DESCRIPTION
**Pull request by markdoerr**
Please add  oso w3id

The Open Science Ontologies - (OSO) are a set of lean, open and modular top-level and domain ontologies designed for interoperable scientific experimentation, fast reasoning and machine learning / AI applications in mind. 

Thanks a lot :)

### CODE REVIEW

**Comment by markdoerr** [comment]
Sorry for the dual PR.
I now separated the two requests by creating a new branch for [lara](https://github.com/perma-id/w3id.org/pull/4327/commits/4f2e7dbcaf108a60ccfbe2e2cef2f7e98c531062) so that oso and lara Pull Requests can be handled separately.

File: lara/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+LARA
+=====
+
+LARA (Laboratory Automation Robotic Assistent) is an open source Research Data Management Suite and 2nd generation Electronic Lab Notebook with the goal to automate most of the data and metadata entering process by providing a rich infrastructure for lab communication, orchestration, data/metadata storage, evaluation and exchange. It utilises Ontology based semantic web technology to make data findable via SPARAQL.
```
Comment (TallTed) [inline lara/README.md:4]:
```suggestion
LARA (Laboratory Automation Robotic Assistent) is an open source Research Data Management Suite and 2nd generation Electronic Lab Notebook with a goal of automating most of the process of entering data and metadata by providing a rich infrastructure for lab communication, orchestration, data/metadata storage, evaluation, and exchange. It uses ontology-based semantic web technology to make data findable via SPARQL.
```

File: oso/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+OSO
+===
+
+The Open Science Ontologies - (OSO) are a set of lean, open and modular top-level and domain ontologies designed for interoperable scientific experimentation, fast reasoning and machine learning / AI applications in mind.
```
Comment (TallTed) [inline oso/README.md:4]:
```suggestion
The Open Science Ontologies (OSO) are a set of lean, open, and modular top-level and domain ontologies designed for interoperable scientific experimentation, fast reasoning, and machine-learning/AI applications.
```

**Comment by dgarijo** [comment]
@markdoerr please have a look at the suggestions by @TallTed

**Comment by markdoerr** [comment]
Thanks, @dgarijo and @TallTed,
for the fast replies - I was busy with writing and refereeing grant proposals ;). 
I will look at everything tonight.

**Comment by markdoerr** [comment]
Thanks a lot to all - it works  :)

**Pull request merged**